src/index.ts

```diff
@@ -168,8 +180,8 @@ export class GoogleToken {
     }
   }
 
-  private async getTokenAsync(): Promise<TokenData> {
-    if (!this.hasExpired()) {
+  private async getTokenAsync(forceRefresh = false): Promise<TokenData> {
```

Comment:
Since this function is private and is only called by `getToken`, does `forceRefresh` here need a default value?
Should also fix coverage.